controllers/article-controllers.js

```diff
@@ -17,4 +27,22 @@ const getArticle = (request, response, next) => {
     });
 };
 
-module.exports = { getArticle, getAllArticles };
+const getArticleComments = (request, response, next) => {
+  const { article_id } = request.params;
+  const promises = [fetchArticleComments(article_id)];
+
+  if (article_id) {
```

Comment:
You can always expect there to be an article_id as it is part of this endpoint and would be caught by an error if not valid